velox/functions/prestosql/DateTimeFunctions.h

```diff
@@ -1770,6 +1779,13 @@ template <typename T>
 struct ToISO8601Function {
   VELOX_DEFINE_FUNCTION_TYPES(T);
 
+  ToISO8601Function() {
+    auto formatter =
+        functions::buildJodaDateTimeFormatter("yyyy-MM-dd'T'HH:mm:ss.SSSZZ");
+    VELOX_CHECK(!formatter.hasError());
+    formatter_ = formatter.value();
+  }
```

Comment:
Does this change belong to this PR?

Reply:
Yes. `functions::buildJodaDateTimeFormatter` now returns `velox::Expected`, so this change is necessary.